docs/rules/no-append-html.md

```diff
@@ -13,6 +13,7 @@ $div.prepend( '<xss>' );
 $div.before( '<xss>' );
 $div.after( '<xss>' );
 $div.replaceWith( '<xss>' );
+$els.add( '<xss>' );
```

Comment:
This file is auto-generated by `npm run doc`. It appears you have manually updated this list, so CI is failing as it doesn't match the generated output (different order).

Reply:
Oops, thanks. There was manual editing involved since `npm run doc` also wanted to make an unrelated change:

```diff
--- a/package-lock.json
+++ b/package-lock.json
@@ -20,7 +20,7 @@
 				"nyc": "^15.1.0"
 			},
 			"peerDependencies": {
-				"eslint": ">=2.3.0"
+				"eslint": ">=6.0.0"
 			}
 		},
 		"node_modules/@aashutoshrathi/word-wrap": {
```